Editors/BackgroundEditor.cpp

```diff
@@ -80,7 +82,8 @@ void BackgroundEditor::on_actionLoadImage_triggered() {
         // QString newData = GetModelData(Background::kImageFieldNumber).toString();
         // TODO: Copy data into our egm and reset the path
         // SetModelData(Background::kImageFieldNumber, lastData);
-      } else qDebug() << "Failed to load gmx Background";
+      } else
+        qDebug() << "Failed to load gmx Background";

```

Comment:
clang-format should have put braces around this; what's up

Reply:
If you know the clang format option to fix it I'll add it but I don't see one